docs/guides/webview-versions.md

```diff
@@ -0,0 +1,190 @@
+[610.3.7.1.9]: https://github.com/WebKit/WebKit/blob/62e4387a5eab36ed075961d9ee9971f8c01a55bd/Source/WebKit/Configurations/Version.xcconfig
+[610.4.3.1.4]: https://github.com/WebKit/WebKit/blob/b152d7889c786689406f203cc4eefea509a90302/Source/WebKit/Configurations/Version.xcconfig
+[610.4.3.1.7]: https://github.com/WebKit/WebKit/blob/248c3283ebdec8bd8ae05d4d1d56390b0da28f27/Sour.3ce/WebKit/Configurations/Version.xcconfig
+[609.3.5.1.3]: https://github.com/WebKit/WebKit/blob/30fc8a44f087596c60e98adb434c0b98eccb61bb/Source/WebKit/Configurations/Version.xcconfig
+[609.4.1.1.1]: https://github.com/WebKit/WebKit/blob/8df64286794c38efa4697b7c24658cb85204a070/Source/WebKit/Configurations/Version.xcconfig
+[609.4.1]: https://github.com/WebKit/WebKit/blob/cb927e6151b5ef49c9ccfb13018f51471f8f1035/Source/WebKit/Configurations/Version.xcconfig
+[609.2.9.1.2]: https://github.com/WebKit/WebKit/blob/ca54d252f3416c3ec64f80a084cb5c4ff7ba24f1/Source/WebKit/Configurations/Version.xcconfig
+[609.1.20.111.8]: https://github.com/WebKit/WebKit/blob/5c90480a38a86464b6b421c2fd28c744b43a4faa/Source/WebKit/Configurations/Version.xcconfig
+[613.3.9.1.5]: https://github.com/WebKit/WebKit/blob/7f88b99524540e94abcdef4d45c1c0324d63fb56/Source/WebKit/Configurations/Version.xcconfig
+
+### Linux (WebKitGTK)
+
+The fractured nature of the Linux ecosystem means it is very hard to compile accurate information about WebKitGTK on the various distros. The table below is a very incomplete list of the most commonly used distributions and their webkit versions. You should always check your distros repositories for up-to-date information.
+
+#### Note on the Ubuntu situation
+
+On Ubuntu 18 through 20 it appears only `amd64` and `i386` machines received subsequent WebKit updates. All other supported arches remained on the initial releases' WebKit version. As you can see this results in really outdated Webkit versionson those machines. Since Ubuntu 22.04 however, it seems all arches receive WebKit updates.
+
+| Distro                                                         | webkitgtk | WebKit    | Safari Equivalent |
+| -------------------------------------------------------------- | --------- | --------- | ----------------- |
+| debian 9                                                       | 2.18      | [605.1.3] | 11.1              |
+| Ubuntu 18.04 (arm64, armhf, ppc64el, s390x)                    | 2.20      | [606.1.4] | 12.0              |
+| Ubuntu 20.04 (arm64, armhf, ppc64el, s390x)                    | 2.28      | [610.1.1] | 14.0              |
+| Ubuntu 18.04 (amd64, i386), centos 8 (non-stream)              | 2.32      | [612.1.6] | 15.0              |
+| debian 10                                                      | 2.34      | [613.1.1] | 15.4              |
+| Ubuntu 20.04 (amd64, i386), Ubuntu 22.04, debian 11, debian 12 | 2.36      | [614.1.6] | TP 140 (16.0)     |
```

Comment:
```suggestion
| Distro                                                         | webkitgtk | WebKit    | Safari Equivalent |
| -------------------------------------------------------------------------- | --------- | --------- | ------------- |
| Ubuntu 18.04                                                                                        | 2.20      | [606.1.4] | 12.0              |
| Debian 9 (with backport), Debian 10                                                    | 2.24      | [???.?.?]  | ??.?              |
| Ubuntu 20.04                                                                                        | 2.28      | [610.1.1] | 14.0              |
| Debian 11, Ubuntu 18.04 (with update), centos 8 (non-stream)          | 2.32      | [612.1.6] | 15.0              |
| Debian 10 (with update)                                                                       | 2.34      | [613.1.1] | 15.4              |
| Debian 11 (with update), Ubuntu 20.04 (with update), Ubuntu 22.04  | 2.36      | [614.1.6] | TP 140 (16.0)     |
```

Debian 9 is a little special. webkitgtk version on release and in the normal update channel are too old, only with the _opt-in_ backports repo teh can get a version >=2.22 (which we set as minimum version in wry, tho later versions may still work idk).
p.s. debian 9 release = 2.16 - debain 9 update = 2.18

@JonasKruckenberg Do you still remember the file you mentioned where you got the webkit version from? Can't seem to find it but i need the webkit version for webkitgtk@2.24

Reply:
webkitgtk@2.24 shipped webkit 608.1.6

Around mid-last year webkit started to switch to GitHub for version tracking, so for newer versions you have to check out the tags on GitHub. For older releases there is https://trac.webkit.org/browser specifically the releases "folder"

Reply:
Thanks! ❤️ i updated the suggestion text.